README.md

```diff
@@ -80,11 +107,16 @@ https://git-scm.com/download/win
 - Install dependencies:
 `yarn install`
 
+- Install json-server:
+`yarn add json-server`
+
```

Comment:
We don't actually need to have this in the README, since its reflected in the `package.json` file and will be installed whenever anyone runs `yarn install`